x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -101,6 +177,44 @@ const getESQuery = (
   // Negative assertions (things that "must not" match)
   const mustNotFilters = buildFiltersForCriteria(negativeCriteria);
 
+  const rangeFilters = [
+    {
+      range: {
+        [sourceConfiguration.fields.timestamp]: {
+          gte: from,
+          lte: to,
+          format: 'epoch_millis',
+        },
+      },
+    },
+  ];
+
+  const sharedSubLevelAggregations = {
+    interval_buckets: {
+      date_histogram: {
+        field: sourceConfiguration.fields.timestamp,
+        fixed_interval: interval,
+      },
+    },
+  };
+
+  const aggregations =
+    groupBy && groupBy.length > 0
+      ? {
+          groups: {
+            composite: {
+              size: 20,
+              sources: groupBy.map((field, index) => ({
+                [`group-${index}-${field}`]: {
+                  terms: { field },
```

Comment:
Currently documents that lack any of the grouping fields are not alerted on, i.e. there's no "everything else" bucket. Is that intentionally left out?